### PR TITLE
Add a task to create an initial version of vscode settings

### PR DESCRIPTION
**Pull request by KevinFairise2**
### What does this PR do?

Add an invoke task that helps to setup `.vscode/settings.json` file with the proper build tags

### CODE REVIEW

File: docs/dev/agent_dev_env.md [hunk 1]
```diff
@@ -259,6 +259,13 @@ pre-commit run ruff --all-files  # run ruff on all files
 
 See `pre-commit run --help` for further options.
 
+### Creating a VSCode settings file
+
+To configure your IDE to work with datadog-agent repository you need to specify build tags in .vscode/settings. Please use the following invoke task to do it the first time:
```
Comment (kayayarai) [inline docs/dev/agent_dev_env.md:264]:
```suggestion
### Creating a Visual Studio Code settings file

To configure your IDE to work with `datadog-agent` repository, specify build tags in `.vscode/settings`. Run the following command to create the file:
```

**Review by kayayarai — APPROVED** [review]
Suggested rewording for style and clarity

File: tasks/build_tags.py [hunk 2]
```diff
@@ -369,3 +369,27 @@ def _compute_build_size(ctx, build_exclude=None, flavor=AgentFlavor.base):
...
+    flavor = AgentFlavor[flavor]
+
+    if targets == "all":
+        targets = build_tags[flavor].keys()
+    else:
+        targets = targets.split(",")
+        if not set(targets).issubset(build_tags[flavor]):
+            print("Must choose valid targets. Valid targets are:")
+            print(f'{", ".join(build_tags[flavor].keys())}')
+            return
```
Comment (CelianR) [inline tasks/build_tags.py:384]: **🥜 nitpick:** ‏Could you use an exception instead ?

File: tasks/setup.py [hunk 3]
```diff
@@ -228,3 +229,57 @@ def download_go_tools(ctx) -> SetupResult:
...
+}}"""
+
+
+@task
+def vscode_settings(_):
+    print(color_message("Creating initial VSCode setting file...", Color.BLUE))
+    if os.path.exists(".vscode/settings.json"):
+        print(color_message("VSCode settings file already exists. Skipping...", Color.ORANGE))
+        user_input = input("Do you want to overwrite it? (y/N)")
+        if user_input.lower() != "y":
```
Comment (CelianR) [inline tasks/setup.py:272]: **🥜 nitpick:** You can use this: https://github.com/DataDog/datadog-agent/blob/2963b0f24cc9cca2ffc12695e891a59ba4fea3cf/tasks/libs/common/user_interactions.py#L4-L19

**Review by CelianR — APPROVED** [review]

**Pull request merged**